content/_data/upgrades/2-277-1.adoc

```diff
@@ -0,0 +1,50 @@
+==== Plugin updates
+
+The configuration user interface transition from HTML `table` layout to HTML `div` layout has required several plugin updates.
+Jenkins administrators should update the installed plugins when they update to Jenkins 2.277.1.
+Some older plugin releases will cause the Jenkins "Configure System" page to fail to save.
+Other older plugin releases will report Java runtime exceptions until they are upgraded to the most recent release.
+
+Popular plugins that need to be updated after upgrading to Jenkins 2.277.1 include:
+
+* plugin:branch-api[Branch API plugin]
+* plugin:envinject[Environment injector plugin]
+* plugin:favorite[Favorite plugin]
+* plugin:github-branch-source[GitHub branch source plugin]
+* plugin:ldap[LDAP plugin]
+* plugin:maven-plugin[Maven integration plugin]
+* plugin:parameterized-trigger[Parameterized trigger plugin]
+* plugin:token-macro[Token macro plugin]
+
+Use the Plugin Manager to update plugins after upgrading to Jenkins 2.277.1.
```

Comment:
shouldn't we linked to the [migration guide](https://www.jenkins.io/doc/developer/views/table-to-div-migration/) and the [dashboard](https://issues.jenkins.io/secure/Dashboard.jspa?selectPageId=20741) in this section too?

Reply:
Different target audience for the migration guide I think, but could a `NOTE: Documentation for plugin maintainers is here:` or similar.